dev/org.eclipse.codewind.openapi.ui/src/org/eclipse/codewind/openapi/ui/Messages.properties

```diff
@@ -48,7 +48,14 @@ WIZARD_PAGE_SELECT_GENERATOR_TYPE=Select a generator type.
 BROWSE_DIALOG_MESSAGE_SELECT_DEFINITION=Select an OpenAPI definition from the project.
 BROWSE_DIALOG_TITLE_SELECT_DEFINITION=Select an OpenAPI definition file.
 
-BROWSE_DIALOG_MESSAGE_SELECT_FOLDER=Select an output folder for the generated files.
-BROWSE_DIALOG_TITLE_SELECT_FOLDER=Select a folder.
+BROWSE_DIALOG_MESSAGE_SELECT_FOLDER=Select an output folder under the current project
```

Comment:
- Add a period at the end of the sentence.
- If possible, replace "under" with "within" if it makes sense to do so. We try to avoid directional words like under, above, etc.

Reply:
Will make those changes.

Reply:
Thanks!